Week1/homework/js-exercises/exercise2.js

```diff
@@ -0,0 +1,2 @@
+'use strict';
+console.log("I'm awesome")
```

Comment:
Don' forget the `;` :)

Reply:
'use strict';
console.log("I'm awesome");